cmd/relayproxy/config/config.go

```diff
@@ -445,6 +451,10 @@ func (c *Config) IsValid() error {
 		}
 	}
 
+	if c.LogFormat != "" && c.LogFormat != "json" && c.LogFormat != "logfmt" {
+		return fmt.Errorf("invalid log format %s", c.LogFormat)
+	}
```

Comment:
**💣 chore:** ‏I have refactor this part to use a switch and lowercase the value.